engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeTypesTest.scala

```diff
@@ -0,0 +1,269 @@
+package org.enso.interpreter.test.instrument
+
+import org.apache.commons.io.output.TeeOutputStream
+import org.enso.interpreter.runtime.EnsoContext
+import org.enso.interpreter.test.Metadata
+import org.enso.polyglot._
+import org.enso.polyglot.runtime.Runtime.Api
+import org.enso.text.editing.model
+import org.enso.text.editing.model.TextEdit
+import org.graalvm.polyglot.Context
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.io.{ByteArrayOutputStream, File}
+import java.nio.file.{Files, Paths}
+import java.util.UUID
+
+@scala.annotation.nowarn("msg=multiarg infix syntax")
+class RuntimeTypesTest
+    extends AnyFlatSpec
+    with Matchers
+    with BeforeAndAfterEach {
+
+  // === Test Utilities =======================================================
+
+  var context: TestContext = _
+
+  class TestContext(packageName: String)
+      extends InstrumentTestContext(packageName) {
+
+    val out: ByteArrayOutputStream    = new ByteArrayOutputStream()
+    val logOut: ByteArrayOutputStream = new ByteArrayOutputStream()
+    val context =
+      Context
+        .newBuilder(LanguageInfo.ID)
```

Comment:
It's probably not necessary and maybe even desirable for these tests, for you don't need to limit the polyglot `Context` to "enso". Now, in `runtime`, you have access to all the languages and instruments that we use everywhere.